pkg/cortex/modules.go

```diff
@@ -577,7 +527,7 @@ var modules = map[ModuleName]module{
 	},
 
 	Ring: {
-		deps:           []ModuleName{Server, RuntimeConfig, MemberlistKV},
```

Comment:
I was under the understanding that modules inherit the dependencies of the their dependencies. I assumed since the API is dependent on the Server, all modules dependent on the API would be dependent on the server?

Reply:
Sure you're right. My feedback was more about making it explicit. Anyway, it's a minor thing and very subjective. I'm resolving the comment.